install/configuring_inventory_file.adoc

```diff
@@ -1589,6 +1589,11 @@ openshift_master_named_certificates=[{"certfile": "/root/STAR.openshift.com.crt"
 openshift_master_overwrite_named_certificates=true
 ----
 
+[IMPORTANT]
+====
+The `cafile` certificate will be imported to `ca-bundle.crt` file on the masters during installation or during redeployment of certificates. The `ca-bundle.crt` is mounted to every pod running in OpenShift. Several OpenShift components will automatically trust the named certificates by default when accessing the `masterPublicURL`. Missing `cafile` in named certificates parameter will affect the functionality of Web Console and several other components.
```

Comment:
```suggestion
The `cafile` certificate is imported to the `ca-bundle.crt` file on the masters during installation or during redeployment of certificates. The `ca-bundle.crt` file is mounted to every pod that runs in {product-title}. Several {product-title} components automatically trust the named certificates by default when they access the `masterPublicURL` endpoint. If you omit the `cafile` option from the certificates parameter, the functionality of Web Console and several other components is reduced.
```

@vwalek, maybe something like this? If there are more appropriate words to drop in, please do, but I think that the sentences should be structured more like this.